src/frontend/components/Header.vue

```diff
@@ -0,0 +1,229 @@
+<script setup lang="ts">
+import { onMounted, onUnmounted, ref, watch } from 'vue'
+import { useRouter } from 'vue-router'
```

Comment:
You can actually remove these imports since they are [auto-imported with Nuxt](https://nuxt.com/docs/4.x/guide/concepts/auto-imports)